### DISCUSSION TITLE
How do I change the theme/css?

### DISCUSSION THREAD

**Discussion by emilyyyylime** [ErikReider/SwayOSD#57] [Q&A]
Currently both the foreground and background colours use dark colours from my theme, making the popups very hard to read. Is it possible to change the theme or css used by SwayOSD?

**Comment by cervyesterday** [comment]
Same here. I tried to write on swayosd-git/src/SwayOsd/data/style/style.css replacing `@theme_fg_color` with rgba() in colors sections but nothing happened.
Any suggestions?

**Comment by drbombz** [comment]
I realize this is an old post, but here's what I did to customize the CSS and use my own CSS:

1. Create DIR to put your custom CSS:

- **Example**: ~/.config/swayosd/style.css

2. Modify your CSS to your liking:

- **Example** (changing BG color/alpha of 'window#osd'):  **background: alpha(@theme_bg_color, 1.0)** --> **background: alpha(#[HEX CODE], 0.4)**

3. Start your 'swayosd-server' with the '-s' switch to use custom style sheet:
- **Eample** (Hyprland): **exec-once = swayosd-server -s ~/.config/swayosd/style.css**

Hope this helps :)
